fulp_modules/features/antagonists/bloodsuckers/code/clans/clan_brujah.dm

```diff
@@ -0,0 +1,105 @@
+		Be wary, as they are ferverous insurgents, rebels, and anarchists who always attempt to undermine local authorities. \n\
+		Their favorite vassal gains the regular Brawn ability and substantially strengthened fists."
+	clan_objective = /datum/objective/brujah_clan_objective
+	join_icon_state = "brujah"
+	join_description = "Gain an enhanced version of the brawn ability that lets you destroy most structures (including walls!) \
+		Rebel against all authority and attempt to subvert it, but in turn <b>break the Masquerade immediately on joining</b>  \
+		and lose nearly all of your Humanity."
+	blood_drink_type = BLOODSUCKER_DRINK_INHUMANELY
+
+
+/datum/bloodsucker_clan/brujah/New(datum/antagonist/bloodsucker/owner_datum)
+	. = ..()
+	owner_datum.special_vassals.Cut() //Removes Discordant Vassal, which is in the list by default.
+	owner_datum.break_masquerade()
+	owner_datum.AddHumanityLost(37.5) // Frenzy at 400
+	bloodsuckerdatum.remove_nondefault_powers(return_levels = TRUE)
+	owner_datum.powers += new /datum/action/cooldown/bloodsucker/targeted/brawn //Doesn't actually give brawn, but makes it unpurchasable.
+	// Copied over from 'clan_tremere.dm' with appropriate adjustment.
+	for(var/datum/action/cooldown/bloodsucker/power as anything in bloodsuckerdatum.all_bloodsucker_powers)
+		if((initial(power.purchase_flags) & BRUJAH_CAN_BUY))
+			bloodsuckerdatum.BuyPower(new power)
+
+/datum/bloodsucker_clan/brujah/spend_rank(datum/antagonist/bloodsucker/source, mob/living/carbon/target, cost_rank, blood_cost)
+	// Give them a quick warning about losing humanity on ranking up before actually ranking them up...
+	var/mob/living/carbon/human/our_antag = source.owner.current
+	var/warning_accepted = tgui_alert(our_antag, \
+		"Since you are part of the Brujah clan, increasing your rank will also decrease your humanity. \n\
+		This will increase your current Frenzy threshold from [source.frenzy_threshold] to \
+		[source.frenzy_threshold + 50]. Please ensure that you have enough blood available or risk entering Frenzy.", \
+		"BE ADVISED", \
+		list("Accept Warning", "Abort Ranking Up"))
+	if(warning_accepted != "Accept Warning")
+		return FALSE
+	return ..()
+
+/datum/bloodsucker_clan/brujah/finalize_spend_rank(datum/antagonist/bloodsucker/source, cost_rank, blood_cost)
+	. = ..()
+	source.AddHumanityLost(5) //Increases frenzy threshold by fifty
+
+///Raise the damage of both of their hands by four. Copied from 'finalize_spend_rank()' in '_clan.dm'
+/datum/bloodsucker_clan/brujah/on_favorite_vassal(datum/antagonist/bloodsucker/source, datum/antagonist/vassal/favorite/vassaldatum)
+	. = ..()
+	var/mob/living/carbon/our_vassal = vassaldatum.owner.current
+	var/obj/item/bodypart/vassal_left_hand = our_vassal.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/vassal_right_hand = our_vassal.get_bodypart(BODY_ZONE_R_ARM)
+	vassal_left_hand.unarmed_damage_low += 4
```

Comment:
Can you make this defines please

Reply:
How might I go about doing that?

Reply:
#define PUNCH_DAMAGE_INCREASE_PER_TIER 4

Reply:
Oh— I thought you meant something _much_ more complex…

In any case, done.